### PR TITLE
Add invert dimmer options for blinds channel

### PR DESCRIPTION
**Pull request by hazzeh**
Add option to invert the dimmer value from blinds.

Signed-off-by: Hans Hazelius <hans@hazelius.se>

### CODE REVIEW

File: addons/binding/org.openhab.binding.zwave/ESH-INF/thing/_channels.xml [hunk 1]
```diff
@@ -230,6 +230,16 @@
                 </options>
                 <limitToOptions>true</limitToOptions>
             </parameter>
+            <parameter name="config_invert_percent" type="boolean">
+                <label>Invert dimmer value</label>
+                <description>Invert the blinds dimmer value</description>
```
Comment (cdjackson) [inline addons/binding/org.openhab.binding.zwave/ESH-INF/thing/_channels.xml]: I think we should remove the word 'dimmer' - it's not inverting a dimmer - it's inverting the percentage (which may, or no, be rendered as a dimmer widget)

**Comment by cdjackson** [comment]
Can I suggest changing the two options to ```config_invert_value``` and ```config_invert_control```?

File: addons/binding/org.openhab.binding.zwave/ESH-INF/thing/_channels.xml (hunk 1 above)
Comment (hazzeh) [inline addons/binding/org.openhab.binding.zwave/ESH-INF/thing/_channels.xml]: Sure. I was unsure what to name it, and just selected dimmer. I will upload a patch.

**Comment by cdjackson** [comment]
Thanks.

**Pull request merged**